docs/service_manager.md

```diff
@@ -0,0 +1,69 @@
+# Defining and Managing Object Database Services
+
+Object Database (ODB) service framework is defined and managed in 
+[the service_manager package](../object_database/service_manager)
+
+## Services
+
+ODB services extends `object_database.service_manager.ServiceBase.Servicebase`.
+The [ServiceBase](../object_database/service_manager/ServiceBase.py) super class
+manages configurations, resources, introspection, and start/stop/setup/teardown.
+
+ODB comes with a set of default services for convenience and testing purposes.
+ - [AwsWorkerBootService](../object_database/service_manager/aws/AwsWorkerBootService.py)
+manages launching and shutdown of AWS ec2 instances.
+ - [ActiveWebService](../object_database/web/ActiveWebService.py) launches a sockets/Cells-based reactive web service
+ - [CellsTestService](../object_database/web/CellsTestService.py) tests display and navigation of cells.
+
+### Task Services
```

Comment:
Just FYI the "Task" infrastructure is no longer used in production.  So we should decide if we want to keep it.